examples/docker-compose.yml

```
@@ -0,0 +1,60 @@
+version: "3.7"
+
+services:
+  httpjs:
+    build:
+      context: ./httpjs
+      dockerfile: Dockerfile
+    container_name: sample-httpjs
+    labels:
+      com.datadoghq.ad.logs: '[{"source": "httpjs", "service": "sample-httpjs"}]'
+    ports:
+      - 8080:8080
+    environment:
+      - DD_ENV=dev
+      - DD_HOSTNAME=local
+      - DD_AGENT_HOST=datadog
+      - DD_TRACE_AGENT_PORT=8126
+
+  apache:
+    build:
+      context: ./apache
+      dockerfile: Dockerfile
+    platform: linux/amd64
+    container_name: sample-apache
+    depends_on:
+      - httpjs
+    image: sample-apache
+    labels:
+      com.datadoghq.tags.env: 'dev'
+      com.datadoghq.tags.service: 'sample-apache'
+      com.datadoghq.tags.version: '0.1.0'
+      com.datadoghq.ad.check_names: '["apache"]'
+      com.datadoghq.ad.init_configs: '[{}]'
+      com.datadoghq.ad.instances: '[{"apache_status_url": "http://%%host%%:81/server-status?auto"}]'
+      com.datadoghq.ad.logs: '[{"source": "apache", "service": "sample-apache"}]'
+    ports:
+      - "8888:8888"
+      - "81:81"
+    environment:
+      DD_AGENT_HOST: datadog
+      DD_TRACE_AGENT_PORT: 8126
+
+  datadog:
+    image: datadog/agent:7
+    container_name: sample-ddagent
+    environment:
+      - DD_API_KEY
+      - DD_SITE=datadoghq.com
```

```suggestion
      - DD_SITE
```
To allow the user to overwrite `DD_SITE`